libs/sdk-embedding/src/iframe/ad.ts

```diff
@@ -176,6 +176,11 @@ export namespace EmbeddedAnalyticalDesigner {
          * Type represent that the remove filter context action is finished
          */
         RemoveFilterContextFinished = "removeFilterContextFinished",
+
+        /**
+         * Type notify KD that the editing insight has been canceled
```

Comment:
I do not think we need to mention KD here, others can use this event as well, right?

Reply:
For the feature https://jira.intgdc.com/browse/GDP-871, KD will open AD overlay. Then the event `VisualizationCancelled` will be sent to KD to close the AD overlay. And It is used internally.

Reply:
Ok, thanks for the explanation :)